charts/karpenter/values.yaml

```diff
@@ -75,6 +75,9 @@ topologySpreadConstraints:
   - maxSkew: 1
     topologyKey: topology.kubernetes.io/zone
     whenUnsatisfiable: ScheduleAnyway
+  - maxSkew: 1
```

Comment:
Can we use hostname antiaffinity instead? If there's 1 node, the scheduler will spread all karpenter pod replicas across all of the nodes, or in this case, just a single node.

Reply:
Sure, we'll update the PR.

Reply:
We might want to let users choose beween `requiredDuringSchedulingIgnoredDuringExecution` and `preferredDuringSchedulingIgnoredDuringExecution`, as different cluster admins might have different preferences.

For example, some clusters bootstrap with one initial node and the admin might want to see their cluster-critical Pods go healthy pronto. Meanwhile, they have a separate mechanism that warns about anti-affinity issues, or even they have their own custom descheduler to take care of that.

Reply:
I was looking to stick with the current defaults where karpenter will still be scheduled on the same node if that's all available.  I think a default rule to naturally spread out is good though.  If users want to take a harder line, would overriding the values be out of order?